### ISSUE TITLE
Clients can spam server with invalid json

### ISSUE THREAD

**Issue by WebFreak001** [AndrewBelt/hack.chat#46]
If many users in the channel execute some code like this:

```
var x="";
for(var i=0;i<65534;i++) x+="[";
setInterval(function() { ws.send(x); }, 0);
```

It'll flood the server with error messages and try/catch isn't the fastest so if many people do it it could probably crash. Its missing POLICE.frisk

**Comment by AndrewBelt** [comment]
Good idea, I should have a frisk before parsing JSON that costs 1 to 0.1 points, instead of adding 1 to each command.

**Comment by AndrewBelt** [comment]
Fixed with 16d59e7f6e2a2da8f8a734000e78ee131f0b4a93
